Makefile

```diff
@@ -45,37 +45,40 @@ endif
 
 all: latest_vc latest_tcc
 ifdef WIN32
-	$(CC) $(CFLAGS) -g -std=c99 -municode -w -o v.exe $(TMPVC)/$(VCFILE) $(LDFLAGS)
+	$(CC) $(CFLAGS) -g -std=c99 -municode -w -o v.exe $(VC)/$(VCFILE) $(LDFLAGS)
+ifdef prod
+	./v.exe -prod self
+else
 	./v.exe self
+endif
 else
-	$(CC) $(CFLAGS) -g -std=gnu11 -w -o v $(TMPVC)/$(VCFILE) $(LDFLAGS) -lm -lpthread
+	$(CC) $(CFLAGS) -g -std=gnu11 -w -o v $(VC)/$(VCFILE) $(LDFLAGS) -lm -lpthread
 ifdef ANDROID
 	chmod 755 v
 endif
+
+ifdef prod
+	./v -prod self
+else
 	./v self
+endif
+
 ifndef ANDROID
 	$(MAKE) modules
 endif
-endif
-ifdef V_ALWAYS_CLEAN_TMP
-	$(MAKE) clean_tmp
 endif
 	@echo "V has been successfully built"
 	@./v -version
 
-#clean: clean_tmp
-#git clean -xf
-
-clean:
```

Comment:
I think it would be good, if there is a `clean: `  make target, that removes the local folder vc/ and does `git clean -xf`, so that the next `make` invocation acts as if the folder was freshly cloned from github, with no remaining local artifacts.

Reply:
OK